internal/infrastructure/cache/memcached/cache.go

```diff
@@ -0,0 +1,65 @@
+package memcached
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/bradfitz/gomemcache/memcache"
+
+	"github.com/andrsj/feedback-service/internal/infrastructure/cache"
+	"github.com/andrsj/feedback-service/pkg/logger"
+)
+
+type Memcached struct {
+	logger        logger.Logger
+	client        *memcache.Client
+	secondsToLive int32
+}
+
+// Check that actual implementation fits the interface.
+var _ cache.Cache = (*Memcached)(nil)
+
+func New(host string, secondsToLive int32, logger logger.Logger) *Memcached {
+	return &Memcached{
+		logger:        logger.Named("memcached"),
+		client:        memcache.New(host),
+		secondsToLive: secondsToLive,
+	}
+}
+
+func (c *Memcached) Get(key string) ([]byte, bool, error) {
+	item, err := c.client.Get(key)
+	if err != nil {
+		if errors.Is(err, memcache.ErrCacheMiss) {
+			c.logger.Info("Cache miss", logger.M{"key": key})
```

Comment:
No need to log here. In production env, there will be a lot of `"Cache miss"` log lines. Better is to use metrics.